server/app/services/applicant/ApplicantData.java

```diff
@@ -79,7 +79,7 @@ public Optional<String> getApplicantName() {
     } catch (NoSuchElementException e) {
       logger.warn(
           "Application {} does not include an applicant name. This is expected for guest users.");
-      return Optional.empty();
+      return Optional.of("Guest");
```

Comment:
Need to revisit, since I think it was still displaying "Logged in as Guest" in the nav bar before